Description: Honor http_proxy= environment variables by default
 In the newly generated dirmngr.conf files only. Existing users behing
 proxies should set honor-http-proxy in $GNUPGHOME/dirmngr.conf,
 see /usr/share/gnupg/dirmngr-conf.skel.
Author: Dimitri John Ledkov <xnox@ubuntu.com>
Bug-Ubuntu: https://bugs.launchpad.net/bugs/1625848

Index: gnupg2-2.2.4/doc/examples/systemd-user/dirmngr.service
===================================================================
--- gnupg2-2.2.4.orig/doc/examples/systemd-user/dirmngr.service
+++ gnupg2-2.2.4/doc/examples/systemd-user/dirmngr.service
@@ -4,5 +4,5 @@ Documentation=man:dirmngr(8)
 Requires=dirmngr.socket
 
 [Service]
-ExecStart=/usr/bin/dirmngr --supervised
+ExecStart=/usr/bin/dirmngr --supervised --honor-http-proxy
 ExecReload=/usr/bin/gpgconf --reload dirmngr
